taxi_service/settings.py

```diff
@@ -56,7 +56,7 @@
 TEMPLATES = [
     {
         "BACKEND": "django.template.backends.django.DjangoTemplates",
-        "DIRS": [],
+        "DIRS": ["templates"],
```

Comment:
The 'DIRS' setting in the TEMPLATES configuration should be a list of directories where Django will search for template files. Ensure that the 'templates' directory exists at the specified path.